lib/api/v3/versions/versions_api.rb

```diff
@@ -42,6 +42,11 @@ class VersionsAPI < ::API::OpenProjectAPI
                                                                     params)
           end
 
+          post &::API::V3::Utilities::DefaultCreate.new(model: Version).mount
+
+          mount ::API::V3::Versions::Schemas::VersionSchemaAPI
+          mount ::API::V3::Versions::CreateFormAPI
+
           route_param :id do
             before do
```

Comment:
As I suggested in https://github.com/opf/openproject/pull/7270, we should avoid `before` blocks in combination with ID params. Instead provide the type of the param through `route_param :id, type: Integer, desc: 'Version ID'` and use `after_validation` block to ensure the parameter has been validated. It does not matter when using a Rails `#find`, but this could have been an added layer to what resulted in the SQLi of AggregatedJournal.